src/behaviors/localization/locales/he.ts

```diff
@@ -1,11 +1,11 @@
-import { IPartialLocaleValues } from "../interfaces/values";
+import { ILocaleValues } from "../interfaces/values";
```

Comment:
Please leave this as `IPartialLocaleValues` so the build doesn't break if we add more values.